app/controllers/AdminUsers/company_values_controller.rb

```diff
@@ -0,0 +1,57 @@
+module AdminUsers
+  class CompanyValuesController < ApplicationController
+    before_action :set_company_value, only: %i[edit update destroy]
```

Comment:
FYI - in real apps callbacks (apart from authentications) are rarely used - they are actually making code a bit harder to read and understand - treat this as an FYI for now - you can refactor it later :)